### ISSUE TITLE
Testing collection

### ISSUE THREAD

**Issue by jennybc** [evanmiller/libxls#21]
The recent reports of "libxls / readxl used to read this xls but now it doesn't" make me wonder about the best way to capture these examples. Most of the files in question are too big to include in readxl's test suite. But it seems like a lost opportunity to not hold on to them in some fashion.

It seems worthwhile to capture them and periodically do a semi-manual test run that verifies: (1) no segfault and (2) reads correct number of rows and columns.

Do you have any thoughts about this?

**Comment by evanmiller** [comment]
A suite of test files makes sense to me! At the moment the libxls repo has exactly 2 test files, which are parsed but not verified.

**Comment by jennybc** [comment]
I discussed this with the tidyverse / r-lib team today, because it happened to be our occasional meeting re: development workflows.

I'm going to devise some place to store "wild" specimens that have been informative. I will then create a system to run readxl on them periodically to check for blatant regressions. I'll keep you posted and will make sure to keep xls separable from xlsx, in case you want to wire something up too.

FYI readxl already has a large set of test sheets:

https://github.com/tidyverse/readxl/tree/master/tests/testthat/sheets

The names alone are rather frightening! But readxl's tests focus on setting sheet geometry and cell/column typing. Not on basic parsing. Most of these files were created by me in Excel, i.e. they are highly domesticated.

Closing and will keep you posted. This is something I plan to do by mid-June (timing is related to an RStudio workweek).
